ethcore/migrations/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "ethcore-migrations"
+version = "0.1.0"
+authors = ["debris <marek.kotewicz@gmail.com>"]
```

Comment:
Don't we use `authors = ["Parity Technologies <admin@parity.io>"]` everywhere?

Reply:
haha, true ;)